### ISSUE TITLE
Documentation for fh-sync-js

### ISSUE THREAD

**Issue by wtrocki** [feedhenry/fh-sync-js#4]
Minimal documentation for development and usage - together with fh-sync cloud.

**Comment by damienomurchu** [comment]
Documentation is already added in PR  fh-sync repo

**Comment by wtrocki** [comment]
Documentation is already added in PR  fh-sync repo:
https://github.com/feedhenry/fh-sync/pull/7

**Comment by damienomurchu** [comment]
Similar work in a doc PR here https://github.com/feedhenry/feedhenry.github.io/pull/11 for feedhenry.org. WIP for the moment, but maybe best approach would be keeping it in fh-sync-js & linking from feedhenry.org instead.

Doc in PR is to provide a complete getting started guide using the fh-sync-js client app & fh-sync server app to walk a user through hooking up sync in JS apps, using those 2 apps as clean examples (all other sync-client apps seems to be muddled in cordova/ angular/ ionic code structures)

**Comment by wtrocki** [comment]
"maybe best approach would be keeping it in fh-sync-js & linking from feedhenry.org instead."

IMHO - that's only one sustainable aproach. Since we have multiple communities for projects we contribute (aerogear, feedhenry, OpenShift, Keycloak or even jenkins) and upstream can be in various places. I will love to see feedhenry.org a gateway to this world, but without duplication of content. Since guides are mostly technical and contain source code they can quickly become outdated. Using links from feedhenry.org will make sure that we referring to the latest and maintained version.

If we trying to improve documentation upstream then we adding much more value than trying to extract some bits of it in different place. 
For some of the projects which already have documentation blogging etc. maybe really god idea as it will organize already existing documentation. Problem with sync is that upstream documentation is not fully created/organized so we may need to focus on this before tackling any other tickets. Without upstream docs we cannot really say that project is open. 

TL;DR - let's allow project maintainers to deal with the content and feedhenry.org facilitate and link to this content. 

PS: We can move this conversation to feedhenry-dev :)

ping @pmdarrow @secondsun for opinions.

**Comment by wtrocki** [comment]
@secondsun - Do you guys think we can merge feedhenry/fh-sync#7 to have temporary place for documentation updates. We can also use separate repository that will contain example app getting started guide etc. - that will be easier to move later to feedhenry.org

**Comment by wtrocki** [comment]
See also: https://github.com/feedhenry/feedhenry-sdk-docs

**Comment by secondsun** [comment]
@wtrocki I think you can merge #7 and keep it as a temorpary place.
